streams/src/main/java/org/apache/kafka/streams/state/internals/AbstractStoreBuilder.java

```diff
@@ -38,8 +38,8 @@ public AbstractStoreBuilder(final String name,
                                 final Serde<K> keySerde,
                                 final Serde<V> valueSerde,
                                 final Time time) {
-        Objects.requireNonNull(name, "name can't be null");
-        Objects.requireNonNull(time, "time can't be null");
+        Objects.requireNonNull(name, "name cannot be null");
```

Comment:
Why we change this?

Reply:
I think it's improves the style. It's internal only anyway.